src/intTest/java/uk/nhs/digital/nhsconnect/nhais/jms/MeshServiceBaseTest.java

```diff
@@ -134,4 +126,17 @@ private MeshMessage deserializeMeshMessage(String meshMessage) {
 
         return dataToReturn.get();
     }
+
+    protected void clearMeshQueue() {
+        await().atMost(10, TimeUnit.SECONDS)
+            .pollDelay(Durations.FIVE_SECONDS)
+            .until(this::isMeshClean);
+    }
+
+    private Boolean isMeshClean() {
```

Comment:
It shouldn't be named with `is` as it's not query, it's command. Name it `acknowledgeAllMessages()`